constants_test.py

```diff
@@ -16,9 +16,9 @@
 
 import os
 
-import constants  # pylint: disable=relative-import
-from core.tests import test_utils  # pylint: disable=relative-import
-import feconf  # pylint: disable=relative-import
+from . import constants
+from . import feconf
+from .core.tests import test_utils
```

Comment:
Do we actually need to add a "." before core.tests? I don't think we do that elsewhere, and we should be consistent throughout the codebase.

Reply:
(Ditto for anything else where the dot doesn't stand on its own.)

Reply:
The linter will throw an error if the dot is not there, though it does accept (and recommend) `oppia.core.tests`.

Reply:
Oh, interesting. But why doesn't it throw an error in other files that do stuff like "from core.tests ..."?

Reply:
Or is this extra "." at the front a convention that we should adopt in the entire application?

Reply:
My guess is that because these files are at root directory which makes this necessary in these files. There are also files in `script/` that are affected, but I think that is also due to a similar issue.

Reply:
Ah, I see, OK.

@apb7 @sanket-mali what do you think? Should this be a general convention for imports or should we have two different standards, one for files in root/scripts and another for other files?

I think I lean towards the former but would appreciate your input. Looking at the "Absolute vs Relative Import" section [here](https://chrisyeh96.github.io/2017/08/08/definitive-guide-python-imports.html) it feels like that's additionally motivated if we want to move to Python 3 at some point (and I think we do, since Python 2 will be deprecated in 2020).

Reply:
@seanlip: Thanks for the link! I went through it and agree that we should adopt the dot convention throughout the code base then. It will be easy moving forward. Thanks!